plugins/labhub.py

```diff
@@ -196,7 +196,7 @@ def unassign_cmd(self, msg, match):
             else:
                 return 'You are not an assignee on the issue.'
 
-    @re_botcmd(pattern=r'mark\s+(wip|pending)\s+https://(github|gitlab)\.com/([^/]+)/([^/]+)/(pull|merge_requests)/(\d+)',  # Ignore LineLengthBear, PyCodeStyleBear
+    @re_botcmd(pattern=r'mark\s+(wip|(?:(pending([-\s]+review)?)))\s+https://(github|gitlab)\.com/([^/]+)/([^/]+)/(pull|merge_requests)/(\d+)',  # Ignore LineLengthBear, PyCodeStyleBear
```

Comment:
Tests and spaces are good, but you are increasing two captured groups. Make them uncaptured groups